javascript/javascript1/week3/seriersDuration.js

```diff
@@ -0,0 +1,52 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+function logOutSeriesText() {
+  const ageInHours = 80 * 365 * 24;
+  for (i = 0; i < seriesDurations.length; i++) {
```

Comment:
Don't forget to declare `i`
`for( let i = 0; ...`

Reply:
:-) thank you Zouhir I will do that